### ISSUE TITLE
Add libevent to oss-fuzz

### ISSUE THREAD

**Issue by rvandermeulen** [libevent/libevent#438]
Google is offering free fuzzing of widely-used open source projects with their OSS-Fuzz initiative.
https://github.com/google/oss-fuzz

Given that libevent is widely used, I think it would make sense to take advantage of that offer and see what they find? :)

**Comment by azat** [comment]
Good idea, I will take a look as soon as 2.1 will be released. Thanks!

**Comment by Google-Autofuzz** [comment]
Hi,
I am on the Google OSS-Fuzz team. I realized that libevent is currently in the process of upstreaming to OSS-Fuzz. Please let me know if have any questions or need any help to do so. We can help to start a PR to OSS-Fuzz. @azat 
Thanks.
